### PR TITLE
#164046263 Users See Article Reading Time

### PR DESCRIPTION
**Pull request by BrianSerem**
**Description**

This Pull request implements users' ability to see the time it takes to read an article
<hr>
**Type of change**

- [x] new feature(non-breaking change which adds functionality).

<hr>

**How has this been tested**

- [x] End to End
- [x] Integration
<hr>

**How to test this manually**
Using Postman, navigate to [https://ah-legion-staging-pr-26.herokuapp.com](https://ah-legion-staging-pr-26.herokuapp.com), use instructions below to test the various endpoints.

##### : Get article reading time
[https://ah-legion-staging-pr-22.herokuapp.com/api/articles/create](https://ah-legion-staging-pr-18.herokuapp.com/api/articles/create)

-Upon retrieval of the article(s), users should be able to see "reading_time" field which is returned inside the article object.
<hr>
**Checklist:**

- [x] This follows all the guidelines for this project.

**Related Stories**
<hr>

[#164046263](https://www.pivotaltracker.com/n/projects/2313280/stories/164046263)

### CODE REVIEW

File: authors/apps/authentication/tests/test_social_auth.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import os
+from decouple import config
```
Comment (abulojoshua1) [inline authors/apps/authentication/tests/test_social_auth.py]: @BrianSerem, why did you add this line of code, it looks redundant here.

**Review by abulojoshua1 — CHANGES_REQUESTED** [review]

File: authors/apps/authentication/tests/test_social_auth.py (hunk 1 above)
Comment (BrianSerem) [reply authors/apps/authentication/tests/test_social_auth.py]: I have removed the redundant import

**Review by RachelleMaina — APPROVED** [review]

**Review by MandelaK — APPROVED** [review]

**Review by Etomovich — APPROVED** [review]

**Review by Etomovich — APPROVED** [review]

**Review by abulojoshua1 — APPROVED** [review]
LGTM
